content/docs/tasks/telemetry/access-log/index.md

```diff
@@ -22,14 +22,20 @@ Edit the `istio` configuration map:
 $ kubectl edit cm istio -n istio-system
 {{< /text >}}
 
-Set the value of `global.proxy.accessLogFile` to "/dev/stdout".
-Be sure to escape quotation marks with backward slashes (`\"`).
+Set the value of `accessLogFile` to "/dev/stdout".
+
+You can choose between JSON and text by setting `accessLogEncoding` to `JSON` or `TEXT`.
 
 You may also want to customize the
-[format](https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log#format-rules) of the access log by editing `global.proxy.accessLogFormat`.
+[format](https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log#format-rules) of the access log by editing `accessLogFormat`.
 
 Save the configuration map and exit the editing mode.
 
+NOTE: All three of these parameters may also be configured via helm values:
```

Comment:
please use a callout: https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#callouts and remove `NOTE`.